src/Runner.Worker/action_yaml.json

```diff
@@ -88,6 +88,7 @@
             "mapping": {
                 "properties": {
                     "using": "non-empty-string",
+                    "env": "runs-env",
```

Comment:
Since, `container-runs-env`'s structure is used by both composite actions and container actions, I decided to change the name so that both types of runs can use it. 

Moreover, to support this change, I made sure to look at all files that contained `container-runs-env` and replace it with `run-env'